### ISSUE TITLE
[CoE Starter Kit - BUG] Power BI Loading blocked by failures with Flow

### ISSUE THREAD

**Issue by Pharosz** [microsoft/coe-starter-kit#4144]
### Describe the issue

I just went trough on configuring the Starter Kit and Audit logs as in the MS documentation, now I am at the part to load Power BI report. When I connect I get the following error: 
Flow: Query 'Flow' (step 'Removed Columns') references other queries or steps, so it may not directly access a data source. Please rebuild this data combination.

 
![PowerBI error](https://user-images.githubusercontent.com/7773165/200803063-0a04ad8f-7ab9-412f-adbb-28cc66b88d17.PNG)

### Expected Behavior

The report loads without error.

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

3.56

### What app or flow are you having the issue with?

Production_CoEDashboard_November2022

### Steps To Reproduce

Starting up the Production_CoEDashboard_November2022 report for the first time.
I enter my production COE env url.
I sign in when prompted.
I get the error when loading as attached.

**Comment by manuelap-msft** [comment]
Hello, we have seen this error before when people were on an old version of Power BI Desktop. What version of Power BI Desktop are you on and are you able to upgrade to the latest?

**Comment by Pharosz** [comment]
Hi Manuela, 
I have downloaded the latest from MS, I have a similar error this time:
Query 'App Connection Reference' (step 'Outer Join') references other queries or steps, so it may not directly access a data source. Please rebuild this data combination.
![image](https://user-images.githubusercontent.com/7773165/200856069-c59be3d8-b99c-4ada-860b-d8f0ae4fc81f.png)

I followed your video tutorial on setup COE (I think it was you), big thank you for that!
My test 2 months ago in a lab environment worked, also the PowerBI, I have the issue now doing the same setup in production with the new COE and PowerBI version.

**Comment by Jenefer-Monroe** [comment]
Can you please close here and then hit refresh? Do you get the same issue on refresh?

**Comment by Pharosz** [comment]
Hi Jenefer! Today morning it is running. I think the only difference is that yesterday I used VPN.

**Comment by wiskaso** [comment]
I figured I would post here even though this is closed because I'm getting the same error.

![image](https://user-images.githubusercontent.com/27371887/203488062-9dd4cb37-9d74-47ab-9abe-01925e93a5cd.png)


I've tried to choose close here and click 'refresh'.. refreshing didn't seem to do anything but the button did go away but when I 'apply changes' I still get this error.

**Comment by manuelap-msft** [comment]
Can you please open a new ticket for this? This helps us correctly triage it and report on how many open items we have to plan our capacity.
